db/seeds.rb

```diff
@@ -1,7 +1,8 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+FactoryGirl.create(:item)
+FactoryGirl.create(:item,
+  item_url: "http://www.ikea.com/us/en/catalog/products/80176284/",
+  title: "HEMNES",
+  subtitle: "Coffee table, black-brown",
+  picture_url: "http://www.ikea.com/us/en/images/products/hemnes-coffee-table-brown__0104030_PE250678_S4.JPG",
+  price: "139.00"
+)
```

Comment:
Similar to my other comment - this wants all of the arguments to be lined up with each other.